falcon-taz/src/test/java/pt/haslab/taz/test/TazTest.java

```diff
@@ -148,6 +156,64 @@ public void loadTraces()
         assertTrue( "#Events notify = " + countNotify + " (expected " + expectedNotify + ")",
                     countNotify == expectedNotify );
 
-        System.out.println( "[OK] Test loadTraces()" );
+        System.out.println( "[OK] loadTracesTest" );
+    }
+
+    @Test
```

Comment:
You don't need this annotation if you add the `test` prefix to the method name.

Example: `testAggregateMessages`